### DISCUSSION TITLE
How to change email adresse for my account if i cannot access the email account ?

### DISCUSSION THREAD

**Discussion by SuperstrongBE** [community/community#57672] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello,
I realize that the email address used as my GitHub login is no longer in use, how can I change it to a valid email address ?

**Comment by neutrinomuon** [comment]
Hi @SuperstrongBE,

I hope you're doing well. Have you tried to change the email address associated with your GitHub account with a login? You can try to enter the GitHub website and log in to your account using your credentials Once logged in, click on your profile picture in the top-right corner of the webpage and select "Settings" from the dropdown menu. And then you can follow these steps:

- **i)** In the left sidebar of the settings page, there exist many options. Click on "Emails" to access the email settings.

- **ii)** You should be able to see your current email address listed under the "Primary email address" section. Click on the "Add email address" button to add a new email address.

- **iii)** Enter your new email, the one you want to use and then click "Add".

- **iv)** GitHub will send a verification email to the new address you have just provided. Open the email and follow the instructions.

- **v)** After verification of your email (new one), return to the GitHub settings page. Under the "Emails" section, you should see the newly added email address listed.

- **vi)** Click on the "Change" button next to the new email address and select "Primary" from the dropdown menu. This will set the new email as the primary one for your GitHub account.

- **vii)** After these steps, you can remove the old email address by clicking "Remove".

and that's it. I hope this helps you. Please let me know

**Reply by ChristianRohde** [reply]
what if Github keeps sending Emails to the old account for verification purpose?

**Reply by sethryanrollins** [reply]
Did you ever get this figured out?

**Comment by halleckcui** [comment]
There is no SETTINGS option in any menu. Please take screenshots if this still exists?!

**Reply by swimswithbricks** [reply]
Yes, what halleckcui said.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @SuperstrongBE ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
